test/integration/relation.js

```diff
@@ -260,6 +280,117 @@ module.exports = function(Bookshelf) {
         equal(_knex.toString(), sql);
       });
 
+      it('should handle a hasOne relation with explicit foreignKeyTarget', function() {
+        const base = new Locale({ id: 1, isoCode: 'en' });
```

Comment:
Use `var` instead of `const`. This is a requirement before Node 0.10 and 0.12 are dropped. I don't see a reason at the moment why should it be done. Only Node 6+ has ES6 near-full support. But people still use Node 4. Meteor just migrated it's core from 0.10 to 4.